### DISCUSSION TITLE
XML/HTML and YAML all support writing literal string without quote, how can KDL (or KDL-like markup language) support this feature?

### DISCUSSION THREAD

**Discussion by xieyuheng** [kdl-org/kdl#316] [Ideas]
The title is the question.

**Comment by zkat** [comment]
This is already discussed over at https://github.com/kdl-org/kdl/discussions/308, and you can make your arguments over there. Last I remember, we decided not to do it?

**Reply by xieyuheng** [reply]
Thanks for your replay and link :)
